pkg/destroy/alibabacloud/alibabacloud.go

```diff
@@ -323,7 +323,7 @@ func (o *ClusterUninstaller) deleteResourceGroup(logger logrus.FieldLogger) (err
 	logger = logger.WithField("name", resourceGroupName)
 	logger.Debug("Searching resource groups")
 
-	response, err := o.listResourceGroups()
+	response, err := o.listResourceGroups(resourceGroupName)
```

Comment:
If the resource group has already been deleted, will the response from `listResourceGroups` be an error or just an empty list?

Reply:
An empty list